API/DTOs/TournamentSubmissionDTO.cs

```diff
@@ -14,18 +14,21 @@ public class TournamentSubmissionDTO
     /// <summary>
     /// The name of the tournament
     /// </summary>
-    public required string Name { get; set; }
```

Comment:
Is this a formatting rule? Does the `required` keyword function the same as `[Required]`? I wouldn't mind using the keyword but that can come later.

Reply:
I had added the `required` keyword quite awhile ago because I was under the assumption that it accomplished the same thing as the `[Required]` attribute but that is not the case. Well, sort of. From my understanding the functional difference for API usage is that using the attribute causes model validation to fail when given an empty string for a value where the keyword does not.

Reply:
good to know